climada/hazard/trop_cyclone.py

```diff
@@ -1311,6 +1319,44 @@ def _bs_holland_2008(si_track: xr.Dataset):
     )
     si_track["hol_b"] = np.clip(si_track["hol_b"], 1, 2.5)
 
+def _bs_holland_2010_v2(si_track: xr.Dataset):
+    """Holland 2010's second version of how to estimate b-value  for sustained surface winds.
+    For version 1 see "_bs_holland_2008"
+
+    The result is stored in place as a new data variable "hol_b".
+
+    Like the original 1980 formula (see `_B_holland_1980`), this approach does also require
+    wind speed measurements, if the wind speed measurements are not available or not  reliable, consider the second
+    option proposed in Holland 2010 (and Holland 2008) implemented as "_bs_holland_2008"
+
+    The parameter applies to 1-minute sustained winds at 10 meters above ground.
+    It is taken from equation (7) in the following paper:
+
+    Holland et al. (2010): A Revised Model for Radial Profiles of Hurricane Winds. Monthly
+    Weather Review 138(12): 4393–4401. https://doi.org/10.1175/2010MWR3317.1
+
+    For reference, it reads
+    b_s = vmax^2 * rho^e / ( 100 * (penv - pcen) )
+    where:
+        rho is the air density ρ (in kg m−3) at the surface level
+        !penv and pcen is assumed to be in hPa in this formula - not Pa, as in our tracks
+
+    Parameters
+    ----------
+    si_track : xr.Dataset
+        Output of `tctrack_to_si`. The data variables used by this function are
+        "cen",  "env", and "vmax". The result is stored in place as a new data
+        variable "hol_b".
+    """
+
+    si_track["hol_b"] = (
+            si_track["vmax"]**2 * RHO_AIR_SURFACE**np.exp(1)
```

Comment:
```suggestion
            si_track["vmax"]**2 * RHO_AIR_SURFACE * np.exp(1)
```

Reply:
To avoid these kinds of mistakes, I strongly suggest to merge the functions `_bs_holland_2010_v2` and `_B_holland_1980` with appropriate optonal parameters to specify the rho, vmax, and clipping to use.